GoogleForCreators/web-stories-wp
# Media: Prevent multiple simultaneous FFmpeg operations for the same file

## Context



## Summary

Improve logic, so that a user can not transcode / mute / trim a single resource at the same time. 

## Relevant Technical Choices



## To-do



## User-facing changes

https://user-images.githubusercontent.com/237508/134203644-550f2c72-f1e7-4024-97e1-e064b2ad1d5f.mov

In this video, you can see the optimization button hide, while the video is muted. 

## Testing Instructions






This PR can be tested by following these steps:

1. Open editor
2. Upload a video ( with sound )
3. Click on Mute video button
4. See trim button / optimize button be hidden. ( see video ) . 


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #8987
